remote/index.html

```diff
@@ -29,25 +29,39 @@
 </nav>
 
 <!-- Begin page content -->
+
 <div class="container">
+<p id="demo">
     <div class="row">
-        <div class="col-md-6 col-md-offset-3"><button id="speak" type="button" class="btn btn-primary btn-lg btn-block glyphicons">Speak!</button></div>
+        <div class="col-md-6 col-md-offset-3">
+            <div class="form-group hidden" id="command">
```

Comment:
Why not just make the text box visible all of the time and put a button next to it that has a microphone?